src/NUnitFramework/nunitlite/TeamCityEventListener.cs

```diff
@@ -104,6 +104,12 @@ public void TestFinished(ITestResult result)
         /// <param name="output">A TestOutput object containing the text to display</param>
         public void TestOutput(TestOutput output) { }
 
+        /// <summary>
+        /// Called when a test produces message to be sent to listeners
```

Comment:
<code>produces *a* message</code>

Change also needed in all ITestListener implementations.